### PR TITLE
Update Deploy VM for Community Build

### PR DESCRIPTION
**Pull request by mukul-seagate11**
-----
[View rendered doc/ProvisionReleaseBuild.md](https://github.com/Seagate/cortx/blob/mukul-seagate11-patch-9-1/doc/ProvisionReleaseBuild.md)

### CODE REVIEW

**Comment by raikrahul** [comment]
>Please use this hostname to make reduce issues further on in the process.


What does this sentence mean?

**Comment by mukul-seagate11** [comment]
> > Please use this hostname to make reduce issues further on in the process.
> 
> What does this sentence mean?

Thanks for pointing out as the statement has been corrected now

File: doc/ProvisionReleaseBuild.md [hunk 1]
```diff
@@ -38,32 +39,50 @@ sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config
...
    [srvnode_default]
    network.data.private_interfaces=ens34, ens35
    network.data.public_interfaces=ens36, ens37
    network.mgmt.interfaces=ens33
    bmc.user=None
    bmc.secret=None
-   storage.cvg.0.data_devices=/dev/sdc
+   
+   #data devices
+   storage.cvg.0.data_devices=/dev/sdc,/dev/sdd,/dev/sde,/dev/sdf
```
Comment (shailesh-vaidya) [inline doc/ProvisionReleaseBuild.md:83]: Have we tested this configuration?

File: doc/ProvisionReleaseBuild.md [hunk 2]
```diff
@@ -76,65 +95,90 @@ sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config
...
 ## Disable the Firewall
 
 ```
-systemctl stop firewalld
-systemctl disable firewalld
+    systemctl stop firewalld
+    systemctl disable firewalld
 ```
 
+## Cleanup temporary repos
```
Comment (shailesh-vaidya) [inline doc/ProvisionReleaseBuild.md:174]: This should be after pre-requisite package installation.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
Please refer - https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/213549354/CORTX+Manual+Deployment+on+Single+Node+VM

File: doc/ProvisionReleaseBuild.md (hunk 1 above)
Comment (mukul-seagate11) [reply doc/ProvisionReleaseBuild.md:83]: yes

File: doc/ProvisionReleaseBuild.md (hunk 2 above)
Comment (mukul-seagate11) [reply doc/ProvisionReleaseBuild.md]: done

File: doc/ProvisionReleaseBuild.md (hunk 1 above)
Comment (mukul-seagate11) [reply doc/ProvisionReleaseBuild.md]: changes to 2 disk as tested as well as ease for the community

**Comment by mukul-seagate11** [comment]
@hessio @justinzw, please review & approve?

**Review by hessio — APPROVED** [review]
Looks great, thanks @mukul-seagate11, great work!

File: doc/ProvisionReleaseBuild.md (hunk 2 above)
Comment (shailesh-vaidya) [reply doc/ProvisionReleaseBuild.md]: Cleaning up repo should be after -  Install Provisioner API and requisite packages steps. Refer - https://github.com/Seagate/cortx-prvsnr/wiki/Deploy-VM-Hosted-Repo

**Comment by mukul-seagate11** [comment]
Duplicate of https://github.com/Seagate/cortx/pull/1063

**Pull request closed without merging**